api.md

```diff
@@ -0,0 +1,168 @@
+## API
+
+All requests can be authenticated by a signed cookie with the session ID.
+
+### POST /session
+Start a new observation session.
+
+Request:
+```
+{
+  name: 'name',
+  observation_method: 'focal_animal',
+  focal_animal_name: 'tarzan',
+  focal_behavior_id: null,
+  session_interval_seconds: '100',
+  session_interval_duration: '15000',
+ 
+  scheme_id: 1234,
+  notes: 'thing',
+
+  // extra metadata
+  location: {
+    lat, long
+  }
+}
```

Comment:
That is, the session and its observations could be coalesced into a single request.